base/REPL.jl

```diff
@@ -958,7 +956,7 @@ type StreamREPL <: AbstractREPL
     waserror::Bool
     StreamREPL(stream,pc,ic,ac) = new(stream,pc,ic,ac,false)
 end
-StreamREPL(stream::IO) = StreamREPL(stream, julia_green, Base.input_color(), Base.answer_color())
+StreamREPL(stream::IO) = StreamREPL(stream, Base.text_color[:green], Base.input_color(), Base.answer_color())
```

Comment:
No, because on most terminals (urxvt, gnome-terminal, cmd etc) bold makes text bold AND also makes the color brighter (the light versions of the color).

![image](https://cloud.githubusercontent.com/assets/1282691/21745184/9f19604a-d525-11e6-9183-95d007262041.png)

The purpose here is to strip the bold part and not change the color, hence it should be `:light_green`.